OpenChart/src/Charting/Key.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+
+namespace OpenChart.Charting
+{
+    /// <summary>
+    /// Represents a key index. The key index refers to the column on a note field.
+    /// </summary>
+    public class Key : IComparable
```

Comment:
could we rename this to KeyIndex to make its purpose clearer?

Reply:
Sure